src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-list/pool-list.component.ts

```diff
@@ -195,6 +205,24 @@ export class PoolListComponent extends ListWithDetails implements OnInit {
       }
     ];
 
+    const yesLabel = $localize`Yes`;
+    const noLabel = $localize`No`;
+    this.filters = [
+      {
+        name: $localize`Show internal pools`,
+        prop: 'pool_name',
+        filterOptions: [noLabel, yesLabel],
```

Comment:
We can have just one filter - show internal pools/hide internal pools.

Reply:
If we are hiding them by default always shall we just rename this as
`Hide internal pools` and then apply `yes` as the default?


Also, admins usually prefer all pools to be shown I suppose to view the pool usage. And they might not like the fact that everytime they visit the page they have to toggle this on. Probably this should be persistent so that it is happy for everyone and don't considered a regression.

Reply:
What I meant was - we should not have yes/no thats extra selection along with selection of "show internal pools"
What we can do is probably have just single filter - "Hide internal pools" or "Show internal pools".

By default hide them I agree too, and then user can always remove filter.
Another thing is to add a tag -"internal" following the names of these pool.

Reply:
See this example https://angular.carbondesignsystem.com/?path=/story/components-table--filtering
Implementation: https://github.com/carbon-design-system/carbon-components-angular/blob/master/src/table/stories/app-model-filter-table.component.ts